src/Document/Content.php

```diff
@@ -42,7 +42,7 @@ class Content
     protected $name;
 
     /**
-     * @PHPCRODM\String
+     * @PHPCRODM\Field(type="string")
```

Comment:
this document should not be used anywhere in the CMF project, I don't think we should update it.